web/src/locales/en.json

```diff
@@ -1,18 +1,17 @@
 {
   "info": {
-    "title": "About the app",
-    "text": "The Electricity Maps app provides real-time insights on web and mobile for citizens, energy experts, NGOs and policymakers promoting a deeper understanding of energy transition challenges.",
-    "open-source-text": "This app is <a href='{{link}}' target='_blank'>Open Source</a> and powered by the community.",
+    "title": "About Electricity Maps",
+    "text": "Electricity Maps is an API platform providing real-time and predictive electricity signals allowing any device to reduce their cost and emissions by informing them about the best time to consume electricity.",
     "version": "App version: {{version}}",
     "share-app": "Share the app:"
   },
   "settings-modal": {
     "title": "Settings"
   },
   "ranking-panel": {
-    "title": "Climate Impact by Area",
-    "subtitle": "Ranked by carbon intensity of electricity consumed (gCO₂eq/kWh)",
-    "search": "Search areas",
+    "title": "Electricity Grid Carbon Emissions",
+    "subtitle": "See where your electricity comes from and how much CO2 was emitted to produce it",
```

Comment:
The language seems great, though we might want to consider adding a period at the end of the subtitle.

Reply:
Feel free to ignore if deemed out of scope since this was pre-existing, but using h1 and h2 makes the title and subtitle compete in my opinion. Maybe we consider turning the subtitle into a p or something:

|before|after|
|---|---|
|<img width="267" alt="Screenshot 2024-12-19 at 10 34 24" src="https://github.com/user-attachments/assets/55cb3d6c-88cc-4ad3-945a-efb03598191f" />|<img width="268" alt="Screenshot 2024-12-19 at 10 24 19" src="https://github.com/user-attachments/assets/ebdbbe7f-a484-4c69-a468-6962c6d83068" />|